tests/env.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+set -e
+
+if [ ! -f "env/bin/activate" ]
+    then
+        python3.7 -m venv env
+fi
+
+source env/bin/activate
+pip install -q -U -r ../tests/requirements.txt
+
```

Comment:
This seems identical to the existing test.sh, could we not reuse that?

Reply:
As discussed, replaced it with `cimetrics_env.sh` that only installs `cimetrics`.